core/domain/summary_services.py

```diff
@@ -131,3 +140,23 @@ def _generate_query(categories):
         })
 
     return results
+
+def get_featured_explorations():
+    """Returns a list of featured explorations."""
+    exp_summary_dict = []
+    exp_summary_dict = exp_services.get_non_private_exploration_summaries()
+    featured_exp_summary_dict = []
+    featured_exp_ids = []
+
+    for key in exp_summary_dict:
+        if exp_summary_dict[key].status == \
+        rights_manager.ACTIVITY_STATUS_PUBLICIZED:
+            featured_exp_summary_dict.append(exp_summary_dict[key])
+            featured_exp_ids.append(exp_summary_dict[key].id)
+
+    if featured_exp_summary_dict:
+        featured_exp_summary_dict = get_displayable_exp_summary_dicts(
+            featured_exp_summary_dict, featured_exp_ids)
+
+    return featured_exp_summary_dict
+    
```

Comment:
Oh! Is it possible to have something like linting cheatsheet? 
So anyone can refer to it if confused or in doubt.